### PR TITLE
codespell: action, config + typos fixed

### PR DESCRIPTION
**Pull request by yarikoptic**
some typos were "functional", possibly fixing an issue  -- please review carefully.

### CODE REVIEW

File: causallearn/search/FCMBased/lingam/CAMUV.py [hunk 1]
```diff
@@ -297,7 +297,7 @@ def get_K(X, width_x):
     return K, Kc
 
 
-def hsic_gam(X=None, Y=None, alph=None, width_x=None, width_y=None, K=None, Kc=None, L=None, Lc=None, mode=None,
+def hsic_gam(X=None, Y=None, alpha=None, width_x=None, width_y=None, K=None, Kc=None, L=None, Lc=None, mode=None,
```
Comment (kunwuz) [inline causallearn/search/FCMBased/lingam/CAMUV.py:300]: Thanks for pointing out these typos! I will comment on some unnecessary (potentially functional) changes like this one. Would you mind removing these before I merge the PR? Thanks!

[unnecessary]

(This does not really mean the change is 'unnecessary'. Just would like to take a safer choice :))

File: causallearn/search/FCMBased/lingam/CAMUV.py [hunk 2]
```diff
@@ -353,7 +353,7 @@ def hsic_gam(X=None, Y=None, alph=None, width_x=None, width_y=None, K=None, Kc=N
         p_value = 1 - gamma.cdf(testStat, al, scale=bet)
         return p_value
 
-    thresh = gamma.ppf(1 - alph, al, scale=bet)[0][0]
+    thresh = gamma.ppf(1 - alpha, al, scale=bet)[0][0]
```
Comment (kunwuz) [inline causallearn/search/FCMBased/lingam/CAMUV.py:356]: [unnecessary]

File: causallearn/search/FCMBased/lingam/hsic2.py [hunk 3]
```diff
@@ -91,7 +91,7 @@ def get_K(X, width_x):
     return K, Kc
 
 
-def hsic_gam(X=None, Y=None, alph=None, width_x=None, width_y=None, K=None, Kc=None, L=None, Lc=None, mode=None,
+def hsic_gam(X=None, Y=None, alpha=None, width_x=None, width_y=None, K=None, Kc=None, L=None, Lc=None, mode=None,
```
Comment (kunwuz) [inline causallearn/search/FCMBased/lingam/hsic2.py:94]: [unnecessary]

File: causallearn/search/FCMBased/lingam/hsic2.py [hunk 4]
```diff
@@ -147,7 +147,7 @@ def hsic_gam(X=None, Y=None, alph=None, width_x=None, width_y=None, K=None, Kc=N
         p_value = 1 - gamma.cdf(testStat, al, scale=bet)
         return p_value
 
-    thresh = gamma.ppf(1 - alph, al, scale=bet)[0][0]
+    thresh = gamma.ppf(1 - alpha, al, scale=bet)[0][0]
```
Comment (kunwuz) [inline causallearn/search/FCMBased/lingam/hsic2.py:150]: [unnecessary]

File: causallearn/search/FCMBased/lingam/CAMUV.py (hunk 1 above)
Comment (yarikoptic) [reply causallearn/search/FCMBased/lingam/CAMUV.py]: done, force pushed

**Review by kunwuz — APPROVED** [review]
Thanks so much!

**Comment by jdramsey** [comment]
Yaroslav Helchenko, are you the Yaroslav Halchecko I know? :-D

Joe (Ramsey)

**Comment by yarikoptic** [comment]
ha - what's up @jdramsey ?  I ran into causal-learn while looking for IMAGES, was not really expecting to see you here in the "Python world" ;)

**Comment by yarikoptic** [comment]
now I spotted https://github.com/py-why/causal-learn#running-tetrad-in-python !

**Comment by jdramsey** [comment]
We're trying our best :) But if you want to give it a shot, that would be
great--your feedback would be wonderful! But don't feel obligated. :)

**Pull request merged**